--- ./build.bash
+++ ../build.bash.old
@@ -86,10 +89,10 @@
 fi
 
 # Actual "go build" call for gocryptfs
-go build "-ldflags=$GO_LDFLAGS" "$@"
+go build -a -tags netgo -ldflags "-w $GO_LDFLAGS -extldflags "-static"" "$@"
 # Additional binaries
 for d in gocryptfs-xray contrib/statfs contrib/findholes contrib/atomicrename ; do
-	(cd "$d"; go build "-ldflags=$GO_LDFLAGS" "$@")
+	(cd "$d"; go build -a -tags netgo -ldflags "-w $GO_LDFLAGS -extldflags "-static"" "$@")
 done
 
 ./gocryptfs -version
